doc/releases/release-notes-3.6.rst

```diff
@@ -546,17 +546,50 @@ Networking
 
 * CoAP:
 
-  * Emit observer/service network events using the Network Event subsystem.
-
+  * Added support for Echo and Request-Tag CoAP options (RFC 9175).
+  * Changed :c:func:`coap_remove_observer` API function return type to bool.
+  * Introduced CoAP Services library, which simplifies implementation of CoAP
+    server functionality.
+  * Updated CoAP server example to use CoAP Services library.
```

Comment:
nit: is it really Services with an s? It seems to be spelled "CoAP service" everywhere.

Reply:
You're right, fixed.